grafana-plugin/src/containers/GrafanaTeamSelect/GrafanaTeamSelect.tsx

```diff
@@ -47,34 +48,37 @@ const GrafanaTeamSelect = observer((props: GrafanaTeamSelectProps) => {
     }
   };
 
+  const content = (
+    <div className={cx('teamSelect', { 'teamSelect--topRight': isNewNavigation() })}>
+      <div className={cx('teamSelectLabel')}>
+        <Label>
+          Select Team{' '}
+          <Tooltip content="The objects on this page are filtered by team and you can only view the objects that belong to your team. Note that filtering within Grafana OnCall is meant for usability, not access management.">
+            <Icon name="info-circle" size="md" className={cx('teamSelectInfo')}></Icon>
+          </Tooltip>
+        </Label>
+        <WithPermissionControl userAction={UserAction.UpdateTeams}>
+          <PluginLink path="/org/teams" className={cx('teamSelectLink')}>
+            Edit teams
+          </PluginLink>
+        </WithPermissionControl>
+      </div>
+      <GSelect
+        modelName="grafanaTeamStore"
+        displayField="name"
+        valueField="id"
+        placeholder="Select Team"
+        className={cx('select', 'control')}
+        value={user.current_team}
+        onChange={onTeamChange}
+      />
+    </div>
+  );
+
   return document.getElementsByClassName('page-header__inner')[0]
-    ? ReactDOM.createPortal(
-        <div className={cx('teamSelect')}>
-          <div className={cx('teamSelectLabel')}>
-            <Label>
-              Select Team{' '}
-              <Tooltip content="The objects on this page are filtered by team and you can only view the objects that belong to your team. Note that filtering within Grafana OnCall is meant for usability, not access management.">
-                <Icon name="info-circle" size="md" className={cx('teamSelectInfo')}></Icon>
-              </Tooltip>
-            </Label>
-            <WithPermissionControl userAction={UserAction.UpdateTeams}>
-              <PluginLink path="/org/teams" className={cx('teamSelectLink')}>
-                Edit teams
-              </PluginLink>
-            </WithPermissionControl>
-          </div>
-          <GSelect
-            modelName="grafanaTeamStore"
-            displayField="name"
-            valueField="id"
-            placeholder="Select Team"
-            className={cx('select', 'control')}
-            value={user.current_team}
-            onChange={onTeamChange}
-          />
-        </div>,
-        document.getElementsByClassName('page-header__inner')[0]
-      )
+    ? ReactDOM.createPortal(content, document.getElementsByClassName('page-header__inner')[0])
+    : isNewNavigation()
+    ? content
     : null;
```

Comment:
I suppose we should render team selector for both: old and new navs

Reply:
It's already handled because it matches the selector `.page-header__inner`